leetcode/array.md

```diff
@@ -0,0 +1,26 @@
+# Array
```

Comment:
Your pull request: LEETCODE-1022 Added reshape-the-matrix


Your commit: LEETCODE-1022 Added reshape-the-matrix

Your commit: LEETCODE-1022 Refactored reshape-the-matrix
Action of your comment must be from ['Added', 'Deleted']